frontend/src/components/applyForm/ApplyForm.tsx

```diff
@@ -60,47 +71,38 @@ const ApplyForm = ({
   }
 
   return (
-    <div className="usa-in-page-nav-container flex-justify">
-      <ApplyFormNav fields={navFields} />
+    <>
       <form
-        className="usa-form usa-form--large flex-1 margin-top-neg-5"
+        className="flex-1 margin-top-2"
         action={formAction}
         // turns off html5 validation so all error displays are consistent
         noValidate
       >
-        <ApplyFormSuccessMessage message={successMessage} />
-        <ApplyFormErrorMessage
-          message={errorMessage}
-          errors={validationErrors}
-        />
-        <FormGroup>{fields}</FormGroup>
-        <p>
+        <div className="display-flex flex-justify">
+          <div>{required}</div>
           <Button
             data-testid="apply-form-save"
             type="submit"
-            onClick={() => window.scrollTo({ top: 0, behavior: "smooth" })}
-            secondary
             name="apply-form-button"
+            className="margin-top-0"
             value="save"
           >
             {pending ? "Saving..." : "Save"}
           </Button>
-          <Button
```

Comment:
Is the submit button being added back in a later PR?

Reply:
No, there is only a "Save" moving forward.